core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/autoscales/AutoscaleV4Endpoint.java

```diff
@@ -129,6 +146,24 @@ void decommissionInternalInstancesForClusterCrn(@PathParam("crn") String cluster
             @RequestBody @NotEmpty List<String> instanceIds,
             @QueryParam("forced") @DefaultValue("false") Boolean forced);
 
+    @DELETE
+    @Path("/stack/stopNodes/crn/{crn}/internal")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = StackOpDescription.STOP_MULTIPLE_INSTANCES_BY_ID_IN_WORKSPACE, produces = APPLICATION_JSON,
+            notes = Notes.STACK_NOTES, nickname = "stopInternalInstancesForClusterCrn")
+    void stopInternalInstancesForClusterCrn(@PathParam("crn") String clusterCrn,
+            @RequestBody @NotEmpty List<String> instanceIds,
+            @QueryParam("forced") @DefaultValue("false") Boolean forced,
+            @QueryParam("scalingStrategy") ScalingStrategy scalingStrategy);
+
+    // TODO CB-14929: Remove this API once done with testing, or publish a quick document somewhere on how the put API can be used
+    @GET
+    @Path("/stack/stopNodes2/crn/{crn}/{userId}")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = "stop_nodes_by_id", produces = APPLICATION_JSON, notes = "blah", nickname = "tmpStopNodes2")
+    String tmpStopNodes2(@PathParam("crn") String crn, @PathParam("userId") String userId,
```

Comment:
same here, we should not merge this test API

Reply:
Same as above about the tmpStart. Should've removed this before the PR.

Reply:
Fixed in the next push. Resolving conversation.